tests/executables_src/testModel.cc

```diff
@@ -195,6 +197,8 @@ BOOST_AUTO_TEST_CASE(testDeviceModuleProxy) {
 BOOST_AUTO_TEST_CASE(testProcessVariableProxy) {
   TestApplication app;
 
+  app.getModel().writeGraphViz("TestApplication.dot");
+
```

Comment:
Is that a left-over or intentional?

Reply:
Thanks, I missed that, it should be removed.